public/src/components/amounts/AmountsVariantEditorRowInput.tsx

```diff
@@ -54,6 +55,7 @@ export const AmountsVariantEditorRowInput: React.FC<AmountsVariantEditorRowInput
   return (
     <div>
       <TextField
+        variant={'standard'}
```

Comment:
keeps the UI as it was before (an underlined field, rather than a field with a box around it)